### PR TITLE
Add Tiwa keynote on frontpage, add code war

### PR DESCRIPTION
**Pull request by jean**

### CODE REVIEW

**Comment by jean** [comment]
Still waiting for official bio for Tiwa

File: site/pages/pycon-thailand.th.rst [hunk 1]
```diff
@@ -91,6 +91,39 @@ meet our keynote speaker
...
+
+       https://www.linkedin.com/in/tiwayork
+
+       ประธานเจาหนาที่บริหาร/เฮดโคช - Kaidee
+       
+       ทิวา ยอรค เฮดโคชแหงทีม Kaidee แพลตฟอรมที่เขาสรางขึ้นมาดวยความเชื่อที่วา “คนไทยทุกคนควรมีแพลตฟอรมสําหรับการซื้อขาย
+       ของออนไลนที่สามารถเขาถึงไดงาย” เขามุงมั่นที่จะพัฒนาตลาดออนไลนแหงนี้ควบคูไปกับการสรางและพัฒนาทีมเทคคนไทยที่มีความ
+       สามารถไมแพทีมเทคระดับโลก
+       
+       Kaidee เริ่มตนขึ้นในป 2011 จากทิวาและทีมอีก 5 คนที่รวมกันสรางแพลตฟอรมนี้ขึ้นมา จนทุกวันนี้กลายเปนที่รูจักและใชงานอยาง
```
Comment (citizen428) [inline site/pages/pycon-thailand.th.rst]: This looks weird here in GitHub, does it look ok on the actual site?

**Review by citizen428 — COMMENTED** [review]
Thanks a lot as always @jean

File: site/pages/pycon-thailand.th.rst (hunk 1 above)
Comment (jean) [reply site/pages/pycon-thailand.th.rst]: Do you mean the blocky spaces?
The rendered version looks fine to me: https://deploy-preview-118--serene-mestorf-5c91d0.netlify.com/
I'll move the bulk of the bio to another page though.

File: site/pages/code-war.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+
+
+Rules of the Game
+=================
+
+Eight teams, onstage knockout rounds of short programming bouts, loud crowd..
```
Comment (citizen428) [inline site/pages/code-war.rst]: Should these two dots be an actual ellipsis (`&hellip;`)?

File: site/pages/code-war.rst [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+We guarantee this won’t be mind twisting problems for algorithm heads. It will
+help to have a team with a mix of skills, a clever and/or intimidating team
+name and vocal cheer squad.
+
+For example: “Here is the complete text of *Pride and Prejudice* from this URL.
+Be the first to print out the 7th and 25th sentences that contain both
+’Elizabeth’ and ‘Darcy’“.
+
+1. One laptop per team and no internet use at any time. You will be live coding.
```
Comment (citizen428) [inline site/pages/code-war.rst]: I thought the word "Internet" is always capitalized in English as it's a proper noun?

https://en.oxforddictionaries.com/definition/internet

File: site/pages/code-war.rst [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+2. Any programming language you want as long it has a large font size. 
+3. Teams are min 3 people, max 5. Don’t have a team? We will find one for you.
+4. Qualification round (7pm sharp). First 8 correct submissions get to form teams.
+5. Head to head knockout rounds to determine the final winner.
+6. After 20min the judge can award the win on points or decide to extend or by dance-off.
```
Comment (citizen428) [inline site/pages/code-war.rst]: I feel likes there's something missing before "dance-off".

**Comment by djay** [comment]
@jean can you put both parties in the menu? Sat Party, Friday warm event?

**Comment by jean** [comment]
@djay done

**Comment by jean** [comment]
@citizen428 I edited the code-war text

File: site/pages/code-war.rst [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+Come along to watch. Come along to compete. Come along to just hangout and meet
+others. Bring your laptop if you want to try the questions from the audience.
+
+**Pizza is provided!**
+
+Rules of the Game
+=================
+
+1. One laptop per team and no internet use at any time. You will be live coding.
+2. Any programming language you want as long it an your IDE has a large font size.
```
Comment (citizen428) [inline site/pages/code-war.rst]: Didn't see this one before, but "as long it an your IDE" seems to be missing bits too.

Comment (jean) [reply site/pages/code-war.rst]: ah!

**Pull request merged**